### PR TITLE
Add encryptSecrets and decryptSecrets route in adminController

### PR DESCRIPTION
**Pull request by jenow**
## What does this PR do ?

Add encryptSecrets and decryptSecrets route in adminController.

### How should this be manually tested?

```http
URL: http://kuzzle:7512/admin/_encryptSecrets
Method: POST
Body:
```

```js
{
  "vaultKey": "your_vault_key",
  "secrets": {
    "aws": {
      "key": "myKey"
    }
  }
}
```

```http
URL: http://kuzzle:7512/admin/_decryptSecrets
Method: POST
Body:
```

```js
{
  "vaultKey": "your_vault_key",
  "secrets": {
    "aws": {
      "key": "a32d94368111ca329958e921a4fe5d70.36d6b839bcfee696b76b62c4de655cd0"
    }
  }
}
```

### Other changes

Make vault throw an exception if `secrets` is not an object.

### CODE REVIEW

File: doc/1/api/controllers/admin/decrypt-secrets/index.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+code: true
+type: page
+title: decryptSecrets
+---
+
+# decryptSecrets
+
+<SinceBadge version="1.10.0" />
+
+Decrypt an object using [Vault](/core/2/guides/essentials/secrets-vault/).
```
Comment (Yoann-Abbes) [inline doc/1/api/controllers/admin/decrypt-secrets/index.md]:
```suggestion
Decrypts an object using [Vault](/core/2/guides/essentials/secrets-vault/).
```

File: doc/1/api/controllers/admin/encrypt-secrets/index.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+code: true
+type: page
+title: encryptSecrets
+---
+
+# encryptSecrets
+
+<SinceBadge version="1.10.0" />
+
+Encrypt an object using [Vault](/core/2/guides/essentials/secrets-vault/).
```
Comment (Yoann-Abbes) [inline doc/1/api/controllers/admin/encrypt-secrets/index.md]:
```suggestion
Encrypts an object using [Vault](/core/2/guides/essentials/secrets-vault/).
```

File: doc/1/api/controllers/admin/decrypt-secrets/index.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+code: true
+type: page
+title: decryptSecrets
+---
+
+# decryptSecrets
+
+<SinceBadge version="1.10.0" />
+
+Decrypts an object using [Vault](/core/2/guides/essentials/secrets-vault/).
```
Comment (Aschen) [inline doc/1/api/controllers/admin/decrypt-secrets/index.md]:
```suggestion
Decrypts an object using [Vault](/core/1/guides/essentials/secrets-vault/).
```

File: doc/1/api/controllers/admin/encrypt-secrets/index.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+code: true
+type: page
+title: encryptSecrets
+---
+
+# encryptSecrets
+
+<SinceBadge version="1.10.0" />
+
+Encrypts an object using [Vault](/core/2/guides/essentials/secrets-vault/).
```
Comment (Aschen) [inline doc/1/api/controllers/admin/encrypt-secrets/index.md]:
```suggestion
Encrypts an object using [Vault](/core/1/guides/essentials/secrets-vault/).
```

File: doc/1/api/controllers/admin/decrypt-secrets/index.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+code: true
+type: page
+title: decryptSecrets
+---
+
+# decryptSecrets
+
+<SinceBadge version="1.10.0" />
+
+Decrypts an object using [Vault](/core/1/guides/essentials/secrets-vault/).
```
Comment (scottinet) [inline doc/1/api/controllers/admin/decrypt-secrets/index.md]:
```suggestion
Decrypts keys stored in the [Vault](/core/1/guides/essentials/secrets-vault/).
```

**Comment by jenow** [comment]
We decided to make the Vault standalone (https://github.com/kuzzleio/kuzzle-vault) therefor it is now useless to have an API endpoint to encrypt/decrypt secrets.

**Pull request closed without merging**